### DISCUSSION TITLE
prisma generate now work

### DISCUSSION THREAD

**Discussion by thomaszdxsn** [prisma/prisma#7855] [Q&A] [answered]
I am following [prisma QuickStart example](https://www.prisma.io/docs/getting-started/quickstart), and can't generate..

When I run `yarn run prisma generate`

The response is always 

    Downloading Prisma engines for darwin [                    ] 0%

After some seconds, it say: " Done in 23.70s."

And nothing change, still not generate client.

My Platform is MacMini M1 chip, MacOS 12.0 Beta

**Comment by thomaszdxsn** [comment, marked as answer]
the download is often stop in 10%~20%, but never prompt me any errors.

**Reply by janpio** [reply]
Please set the `DEBUG=*` env var and post the full output you are getting.

Can you access https://binaries.prisma.sh/ on your computer and e.g. download https://binaries.prisma.sh/2.0.x/7d77a7cd570c9dedb4cdf31a1cbec615cf2e1eda/darwin/introspection-engine.gz?

**Reply by thomaszdxsn** [reply]
After many tries, finally it download.

Maybe this url is block by my country's GFW

**Reply by janpio** [reply]
Yes, that is unfortunately possible. We are working on getting infrastructure up that is not blocked, but it is proving pretty hard without a real investment in time and money.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
